### DISCUSSION TITLE
1.19.2ic2：b27b4f8 生存第一天bug上传

### DISCUSSION THREAD

**Discussion by Dur-BB** [IndustrialCraftMC/IndustrialCraft2#111] [【漏洞反馈】Bugs Report]
大量的合成表在jei查不到；电力喷气背包还没得；橡胶树苗打一下就碎；粗锡不可以用高炉烧；橡胶树只可以提取一次然后就不会再长橡胶了；**超级严重bug：打粉机（还有电炉）在工作中途替换材料以后会扣除替换后材料数量但是生成的是替换前的材料的打粉（电炉燃烧）产物**；成就系统有一些还是乱码；  还有一点建议就是铁锭和锡锭太像了有点，建议还是让锡锭更透亮一些或者深色一点

**Comment by AbbottMc** [comment]
橡胶树只可以提取一次然后就不会再长橡胶了这个问题确认存在吗？一般情况下平均需要八分钟左右才能重新生成，如果运气不好的话，可能时间会更长些

**Reply by Dur-BB** [reply]
这个确定有了，我刚才试了一下，大概有二十分钟也没长出来，在有树叶和没树叶都长不出来，还有就是橡胶树苗和导线破坏不掉落

**Reply by AbbottMc** [reply]
已修复

**Comment by AbbottMc** [comment]
进度系统会在后续内容迁移完成后统一重构
